### PR TITLE
remove deploy/releases from the repository and upload assets at release time

### LINKED ISSUE #55: Document or script release specific changes
Missing release artifacts are easy when the process is not documented, scripted, or automated. The current release of packet-ccm is v1.0.1, however the latest [deployment recommended is v1.0.0](https://github.com/packethost/packet-ccm/blob/v1.0.1/README.md#deploy-project-and-api).  The [`deploy/release` path](https://github.com/packethost/packet-ccm/tree/v1.0.1/deploy/releases) was not updated in the latest release.

Update the documentation or scripts to make release deployment a better process for the engineer performing the steps.

### PR DESCRIPTION
**Pull request by deitch**
Closes #55

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,11 +93,10 @@ packet-cloud-config   Opaque                                1         2m
 
 ##
```
Comment (gianarb) [inline README.md:94]: This looks like a missed title

Comment (deitch) [reply README.md:94]: It is, but it was there before. Hold on, I will fix it.

Comment (deitch) [reply README.md]: Fixed. Nice catch.

**Comment by deitch** [comment]
CI is stuck, but it just is the report part, not anything related to build, so I am merging it in.

**Pull request merged**